### PR TITLE
New: "Dressing room"

### PR DESCRIPTION
**Pull request by lowercase-donkey**
Finally all the widgetting pays off.

Nice to have the full descriptions near each item, and we now have a LOT of room to add clothing without slaveInteract getting out of hand.

### CODE REVIEW

**Comment by freecitiesbandit** [comment]
This is really neat. You've been killing it with all your additions lately.

**Comment by lowercase-donkey** [comment]
Thanks! :blushes:

**Comment by lowercase-donkey** [comment]
Ready for merge.  I'll probably keep tinkering, but the major stuff is here.

**Comment by pregAnon** [comment]
One comment.  The wardrobe was designed to have a built in dressing area along with being clothing storag, so you might want to keep that in mind with describing where the dressing room more or less is.

**Comment by lowercase-donkey** [comment]
I hadn't thought of that, thank you.
Looking into it, yeah, this would be the wardrobe room we are describing here.  Looks like there's a few references to it.

I'll make a fix.

**Comment by lowercase-donkey** [comment]
ugh, stupid rebase.

**Comment by lowercase-donkey** [comment]
I'm calling attention to this here because it's easy to see in this build and screen, but Unnur  needs your help.

Unner has an $activeSlave.clit of 0.

Remote surgery sees this a small clit, and allows surgery on it:

$pronounCap has
<<if $activeSlave.clit == 0>>
	a small clit.
<<elseif $activeSlave.clit == 1>>
	a big clit.
<<elseif $activeSlave.clit == 2>>
	a huge clit.
<<elseif $activeSlave.clit > 2>>
	 an enormous clit.
<</if>>
<<if $activeSlave.clit > 0>>
	<<if $activeSlave.indentureRestrictions < 2>>
	[[Apply clitoral reduction|Surgery Degradation][$activeSlave.clit = 0,$cash -= $surgeryCost, $activeSlave.health -= 10,$surgeryType = "clitoral reduction"]]
	<</if>>
<</if>>
<<if ($activeSlave.clit < 3) && ($surgeryUpgrade == 1)>>
	<<if $activeSlave.indentureRestrictions < 1>>
	<<if $activeSlave.clit > 0>> | <</if>>
	 [[Increase clit|Surgery Degradation][$activeSlave.clit += 1,$cash -= $surgeryCost, $activeSlave.health -= 10,$surgeryType = "clitoral enlargement"]]
	<</if>>
<</if>>
<</if>>

However, <<VaginaDescription>> sees it as 
`The base of her cock is located where the clitoris would be on a normal woman.`

Thoughts?

[free-cities-20170713-225141.zip](https://github.com/Free-Cities/Free-Cities/files/1147208/free-cities-20170713-225141.zip)

**Comment by freecitiesbandit** [comment]
Looking forward to seeing this get merged. I've made a small list of clothes and tattoos I'd like to work on and get added.

File: src/uncategorized/descriptionWidgetsStyle.tw [hunk 1]
```diff
@@ -0,0 +1,3385 @@
...
+		<<case "a penitent nuns habit">>
+			is curled into long flowing locks barely hidden by $possessive surplice.
+		<<case "a slave gown" "a ball gown" "a halter top dress">>
+			is curled into long flowing locks, almost reaching the ground.
+		<<case "slutty business attire">>
+			is curled into long flowing locks, secured by gaudy gold clasps.
+		<<case "nice business attire">>
+			is curled into long flowing locks and tied up so $pronoun can do business without them getting in the way.
+		<<case "a slutty nurse outfit" "a nice nurse outfit">>
+			is curled into long flowing locks, secured by white cloth ties emblazoned with little
```
Comment (Folant) [inline src/uncategorized/descriptionWidgetsStyle.tw]: ```$possesive``` --> ```$possessive```

Comment (Folant) [inline src/uncategorized/descriptionWidgetsStyle.tw]: ```$possesive``` --> ```$possessive```

Comment (Folant) [inline src/uncategorized/descriptionWidgetsStyle.tw]: ```$possesive``` ---> ```$possessive```

**Comment by Folant** [comment]
https://github.com/lowercase-donkey/Free-Cities/blob/8131b327c8e468d57b3cbde8832602941d2c24a4/src/uncategorized/descriptionWidgetsStyle.tw#L2606
```$possesive``` ---> ```$possessive```

File: src/uncategorized/descriptionWidgetsStyle.tw (hunk 1 above)
Comment (Folant) [inline src/uncategorized/descriptionWidgetsStyle.tw]: 1

Comment (Folant) [inline src/uncategorized/descriptionWidgetsStyle.tw]: 2

Comment (Folant) [inline src/uncategorized/descriptionWidgetsStyle.tw]: 3

**Comment by Folant** [comment]
Lines 2619,2632,2726 ```$posessive``` --> ```$possessive```
Damn github-review just don't work

**Comment by lowercase-donkey** [comment]
Oh I believe you @Folant , that's easily my commonest misspelling :(

**Comment by freecitiesdev** [comment]
I'm reasonably certain I resolved the couple of minor conflicts correctly, but please backcheck me.

**Pull request merged**